zeekscript/script.py

```diff
@@ -55,7 +57,8 @@ def parse(self):
         except OSError as err:
             raise FileError(str(err)) from err
 
-        self.ts_tree = Parser().parse(self.source)
+        parser = tree_sitter.Parser(tree_sitter.Language(tree_sitter_zeek.language()))
```

Comment:
This API is only available in the current version. Since [`tree-sitter` stopped publishing versions for python-3.8](https://pypi.org/project/tree-sitter/#files) for python-3.8 we would get an older version which provides an incompatible API. For this reason I dropped support for python-3.8 in the GH action.